nutils/sample.py

```diff
@@ -755,9 +753,8 @@ def eval_integrals_sparse(*integrals: evaluable.AsEvaluableArray, **arguments: M
   results : :class:`tuple` of arrays and/or :class:`nutils.matrix.Matrix` objects.
   '''
 
-  integrals = tuple(integral.as_evaluable_array.assparse for integral in integrals)
-  with evaluable.Tuple(tuple(integrals)).optimized_for_numpy.session(graphviz=graphviz) as eval:
-    return eval(**arguments)
+  warnings.deprecation('sample.eval_integrals_sparse is deprecated, use function.eval_sparse instead')
```

Comment:
`function.Array.eval` still uses `sample.eval_integrals`.

Reply:
..which is not deprecated? Not sure that I see the issue. Anyway, I moved eval_sparse to evaluable now, and added function.eval in a separate commit.

Reply:
Ah, I missed that `sample.eval_integrals` was not deprecated. Shouldn't we deprecated `sample.eval_integrals` as well?

Reply:
The thing is this (and Sample.integrate) is where _convert is called, so it's not the same as the new function.eval. It should go eventually but I don't think we can deprecate it yet.

Reply:
Ok.